lib/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart

```diff
@@ -32,10 +32,12 @@ class DisplayMapFromMobileMapPackageSample extends StatefulWidget {
 
 class _DisplayMapFromMobileMapPackageSampleState
     extends State<DisplayMapFromMobileMapPackageSample> {
+  // creates a controller for the map view.
```

Comment:
```suggestion
  // create a controller for the map view.
```
These comments are good - just make the verb more like "do this" than "this does this" so it reads like instructions, if that makes sense.